### DISCUSSION TITLE
[Docs]: FAQ

### DISCUSSION THREAD

**Discussion by igor-hara** [amannn/next-intl#827] [Q&A]
### Link to page

https://next-intl-docs.vercel.app/docs/faq

### Describe the problem

Hello next/intl Team!

Is there a way to not use a `src` folder for app structure?

and how to add a Language Switcher dropdown with more than 2 languages

**Comment by amannn** [comment]
> Is there a way to not use a `src` folder for app structure?

Yep, this is supported.



> and how to add a Language Switcher dropdown with more than 2 languages

Please see https://next-intl-docs.vercel.app/docs/routing/navigation#userouter.


I'll move this to a disucssion since this seems to be a usage question.
